### PR TITLE
Gnutls update to 3.6.14, nettle update to 3.6

### PR DESCRIPTION
**Pull request by FredDodd6**
Gnutls update requires the nettle recipe to be updated.
Gnutls from 3.4.17 to 3.6.14.
nettle from 3.3 to 3.6.

### CODE REVIEW

File: recipes/gnutls/3.6.14/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  name: gnutls-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://gnutls.org/
+  license: LGPL
+  summary: "The GnuTLS Transport Layer Security Library."
+
+build:
+  number: 3
```
Comment (kjsanger) [inline recipes/gnutls/3.6.14/meta.yaml:15]: When we bump a version number, builds should be set to `0` (the first build of the new version)

File: recipes/nettle/3.6/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  name: nettle-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://www.lysator.liu.se/~nisse/nettle/
+  license: GPL2
+  summary: A low-level cryptographic library
+
+build:
+  number: 5
```
Comment (kjsanger) [inline recipes/nettle/3.6/meta.yaml:14]: When we bump a version number, builds should be set to `0` (the first build of the new version)

**Comment by dkj** [comment]
Marking "work in progress" as we need to fix our library ABI versions and dependencies before merging...

File: recipes/gnutls/3.6.14/build.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -ex
+
+n="$CPU_COUNT"
+PKG_CONFIG="/usr/local/pkg-config-0.28"
```
Comment (kjsanger) [inline recipes/gnutls/3.6.14/build.sh:6]: Is this going to be there on the build host?

**Comment by kjsanger** [comment]
Has this library changed ABI, or one of its dependencies?

File: recipes/gnutls/3.6.14/build.sh (hunk 3 above)
Comment (FredDodd6) [reply recipes/gnutls/3.6.14/build.sh:6]: No, that probably shouldn't included.

**Pull request closed without merging**